src/components/Disputes/panels/CommitPanel.js

```diff
@@ -271,7 +272,9 @@ const RevealService = React.memo(function RevealService({
         By enabling this feature you trust Aragon One to reveal your vote on
         your behalf in this and following disputes. You can always turn off this
         service later if you choose.
-        {/* TODO: Add ref:  <Link>Learn more</Link>. */}
+        <Link href="https://help.aragon.org/article/43-dispute-lifecycle#onetime">
+          Learn more.
+        </Link>
```

Comment:
```suggestion
        <Link href="https://help.aragon.org/article/43-dispute-lifecycle#onetime">
          Learn more
        </Link>
        .
```

Super tiny nitpick, but we place periods out of links :).